src/schema/v2/artist/meta.ts

```diff
@@ -1,45 +1,86 @@
 import { stripTags, truncate, markdownToText } from "lib/helpers"
-import { compact } from "lodash"
-import { GraphQLString, GraphQLObjectType, GraphQLFieldConfig } from "graphql"
+import {
+  GraphQLString,
+  GraphQLObjectType,
+  GraphQLFieldConfig,
+  GraphQLEnumType,
+} from "graphql"
 import { ResolverContext } from "types/graphql"
 
 export const metaName = (artist) => {
   if (artist.name) return stripTags(artist.name)
   return "Unnamed Artist"
 }
 
+const ArtistPageEnumType = new GraphQLEnumType({
+  name: "ArtistPage",
+  values: {
+    ABOUT: { value: "ABOUT" },
+    ARTWORKS: { value: "ARTWORKS" },
+    AUCTION_RESULTS: { value: "AUCTION_RESULTS" },
+  },
+})
+
 const ArtistMetaType = new GraphQLObjectType<any, ResolverContext>({
   name: "ArtistMeta",
   fields: {
     description: {
       type: GraphQLString,
-      resolve: (artist) => {
+      resolve: ({ artist, page }) => {
         const blurb = artist.blurb.length
           ? markdownToText(artist.blurb)
           : undefined
 
-        const description = compact([
-          `Find the latest shows, biography, and artworks for sale by ${metaName(
-            artist
-          )}`,
-          blurb,
-        ]).join(". ")
-        return truncate(description, 157)
+        switch (page) {
+          case "ABOUT":
+            return truncate(
+              `Explore ${metaName(
+                artist
+              )}'s biography, achievements, artworks, auction results, and shows on Artsy. ${blurb}`,
+              157
+            )
+          case "ARTWORKS":
+            return truncate(
+              `Discover and purchase ${metaName(
+                artist
+              )}’s artworks, available for sale. Browse our selection of paintings, prints, and sculptures by the artist, and find art you love. ${blurb}`,
```

Comment:
Why I think this should have some specs:

```tsx
`Foo. ${null}`
// => 'Foo. null'
`Foo. ${undefined}`
// => 'Foo. undefined'
`Foo. ${null ?? ''}`
// => 'Foo. ' // Note the trailing space
```

Reply:
specced it and changed `undefined` to `""` &  made it so the space only shows up if there's a blurb.

I have some Qs, though:
* Google will automatically truncate the length of the element if it's too long, so given that the blurb is supposed to be short anyway, does it matter if we truncate or not?
* inversely, why include the blurb/bio at all, if it's getting truncated? like, will that really impact SEO?

(I'll raise in #grow)